aws/aws.go

```diff
@@ -0,0 +1,42 @@
+package aws
+
+import (
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+var regions []string
+
+// GetAllResources - Lists all aws resources
+func GetAllResources() []string {
+	regions = []string{
+		"us-east-1", "us-east-2", "us-west-1", "us-west-2", "ca-central-1",
+		"eu-west-1", "eu-central-1", "eu-west-2", "ap-southeast-1", "ap-southeast-2",
+		"ap-northeast-2", "ap-northeast-1", "ap-south-1", "sa-east-1",
+	}
```

Comment:
I think you can get the list of regions programmatically from this API: https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/